src/systems/systemDialog.component.tsx

```diff
@@ -361,9 +313,9 @@ const SystemDialog = React.memo((props: SystemDialogProps) => {
             Save
           </Button>
         </Box>
-        {formError && (
-          <FormHelperText sx={{ marginTop: 4 }} error>
-            {formError}
+        {errors.root?.formError && (
+          <FormHelperText sx={{ marginBottom: '16px' }} error>
```

Comment:
```suggestion
          <FormHelperText sx={{ marginBottom: 4 }} error>
```

Reply:
This can be address  in the audit

Reply:
Sorry i didn't realise this was complete wrong. I've change it back to this  

`<FormHelperText sx={{ marginTop: 4 }} error>`